wiki/Game_modifier/No_Fail/zh-tw.md

```diff
@@ -0,0 +1,37 @@
+---
+stub: true
+tags:
+  - no fail
+  - NF
+  - mod
+  - game modifier
+  - 遊戲 Mod
+---
+
+# No Fail (mod)
+
+![NF Mod 圖示](/wiki/shared/mods/NF.png "No Fail (NF) Mod 圖示")
+
+*若要查看 Mod 列表，請見：[遊戲 Mod](/wiki/Game_modifier)*
+
+## 關於
+
+- 別名：NF
+- 類型：降低難度
+- 成績加乘：0.50 倍
+- 預設快捷鍵：`W`
+- 註解：`不論如何，你都不會失敗。`
+- 適用的遊戲模式：![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## 說明
+
+啟用 No Fail [Mod](/wiki/Game_modifier) 後，即使玩家的[血條](/wiki/Glossary/Health_bar)歸零，仍不會強制終止遊戲 (失敗)。但是，如果玩家完全沒有獲得任何分數，結束後依舊*會*算為失敗。
```

Comment:
missing bold of `No Fail`

missing link to `/wiki/Beatmap`

Reply:
I skipped the beatmap because translate it would make the sentence looks weird